app/models/annotation.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Annotation < ApplicationRecord
+  SUPPORTED_ANNOTATIONS = %w[managed].freeze
+
+  belongs_to :annotated, polymorphic: true, required: true, inverse_of: :annotations
```

Comment:
Just quoting what RuboCop says:
```
RuboCop: You specified `required: true`, in Rails > 5.0 the required option is deprecated and you want to use `optional: false`. In most configurations, this is the default and you can omit this option altogether [Rails/BelongsTo]
```

Reply:
Thanks, for some reason my Rubocop doesn't warn about this. Fixed in https://github.com/3scale/porta/pull/3857/commits/05ddcb66fa3029854354e9aae207886d04aade0b